flume/doc/design/architecture.md

```diff
@@ -1 +1,77 @@
-TBD
+#Cygnus architecture
```

Comment:
Maybe the document should include the how Cygnus works in HA, from a point of view of architecture and sequence diagrams?